tsconfig.json

```
@@ -24,7 +24,7 @@
     "node_modules"
   ],
   "include": [
-    "./src/index.ts",
-    "./types"
+    "./types",
+    "./src"
```

```suggestion
    "./src",
    "./types"
```

uber-nit: Alphabetical order